chain/chain/src/types.rs

```diff
@@ -567,12 +567,6 @@ pub trait RuntimeAdapter: Send + Sync {
         state_root: &StateRoot,
     ) -> bool;
 
-    fn chunk_needs_to_be_fetched_from_archival(
```

Comment:
This is moved to chunks/src/logic.rs; it makes no sense to be part of the RuntimeAdapter.